curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/step-061.md

```diff
@@ -60,7 +60,7 @@ assert.equal(new __helpers.CSSHelp(document).getStyle('button')?.fontSize, '23px
 You should give `button` a `background` of `#d0d0d5`.
 
 ```js
-assert.equal(new __helpers.CSSHelp(document).getStyle('button')?.background, 'rgb(208, 208, 213)');
+assert.include(['rgb(208, 208, 213)','rgb(208, 208, 213) none repeat scroll 0% 0%'], new __helpers.CSSHelp(document).getStyle('button')?.background);
```

Comment:
```suggestion
assert.equal(new __helpers.CSSHelp(document).getStyle('button')?.backgroundColor, 'rgb(208, 208, 213)');
```

Reply:
Was this one intentionally omitted?

Reply:
No, this change is fine by me. Did I not commit it properly? The "Commit suggestion" button is grayed out so I thought I did but perhaps I didn't do something correctly?

Reply:
I'm not sure why it was grayed out, it doesn't appear to be committed. If committing suggestion button still won't want to work, you can make this change manually on local branch (after pulling two added commits) and then push new commit to remote.

Reply:
Don't know what happened but as soon as I came back in to look at your last comment the "Commit suggestion" button was active.